src/pages/404.tsx

```diff
@@ -0,0 +1,156 @@
+import { Box, Flex, Image, Text, useMediaQuery } from '@chakra-ui/react';
+
+import Footer from '@/uikit/Footer';
+import Header from '@/uikit/Header';
+
+function DescriptionMobile() {
+  return (
+    <Text as='h3' fontWeight='light'>
+      Whoops, you’ve wandered far, even Pluto is in your rearview! Seems like
+      the page you’re looking for has drifted off-course
+    </Text>
+  );
+}
+
+function DescriptionDesktop() {
+  return (
+    <>
+      <Text as='h3' fontWeight='light'>
+        Whoops, you’ve wandered far, even Pluto is in your rearview!
+      </Text>
+      <Text>Seems like the page you’re looking for has drifted off-course</Text>
+    </>
+  );
+}
+
+export default function Page404() {
+  const isMobile = useMediaQuery('(max-width: 768px)')[0];
+  const isDesktop = useMediaQuery('(min-width: 1024px)')[0];
+
+  return (
+    <Flex flexDirection='column'>
+      <Box w='full' position='absolute' zIndex={15}>
+        <Header />
+      </Box>
+      <Flex
+        justifyContent='center'
+        alignItems='center'
+        flexDirection='column'
+        w='full'
+        h='100vh'
+        background='black'
+        position='relative'
+        overflow='hidden'>
+        <Flex
+          zIndex={10}
+          color='white'
+          alignItems='center'
+          flexDirection='column'
+          px='10'>
+          <Text
+            as='h1'
+            fontWeight='bold'
+            fontSize={{ base: '128px', lg: '192px' }}>
+            404
+          </Text>
+          <Box
+            fontSize={{ base: '16px', md: '24px', lg: '32px' }}
+            textAlign={{ base: 'center' }}>
+            {isMobile && <DescriptionMobile />}
+            {isDesktop && <DescriptionDesktop />}
```

Comment:
https://github.com/mocha-bot/land/pull/45/files#r1796400491

With this suggestion, you don't need separate mobile and desktop component anymore, this is applied to other component as well

Reply:
updated